pages/Product Page/products.css

```diff
@@ -0,0 +1,55 @@
+.main-container {
+	margin-top: 6rem;
+	display: grid;
+	grid-template-columns: 1fr 3fr;
+}
+
+
+.filter-container {
+	margin-top: 10rem;
+	margin-bottom: 10rem;
+	margin-left: 2rem;
+	background-color: var(--bg-color-1);
```

Comment:
CSS variable naming is not good

Reply:
Will try to make it better thanks for pointing